tests/db/test_base.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+import pytest
+import sqlite3
+
+from monkeytype.db.base import CallTraceStoreLogger
+from monkeytype.db.sqlite import (
+    create_call_trace_table,
+    SQLiteStore,
+)
+from monkeytype.tracing import trace_calls
+from _pytest.monkeypatch import MonkeyPatch
+
+
+def func(a, b):
+    pass
+
+
+def func2(a, b):
+    pass
+
+
+def func3(a, b):
+    pass
+
+
+@pytest.fixture
+def logger() -> CallTraceStoreLogger:
+    conn = sqlite3.connect(':memory:')
+    create_call_trace_table(conn)
+    return CallTraceStoreLogger(SQLiteStore(conn))
+
+
+def test_round_trip(logger):
+    from types import ModuleType
+    module = ModuleType('__main__')
```

Comment:
I don't think any of this business with constructing a fake module is really necessary, is it? All that matters is setting the `__module__` attribute on one of your three functions to the string `__main__`.